File: C++/A/1941A-rudolfandticket.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int t, n, m, k;
    cin >> t;
    while (t--)
    {
        int sum = 0;
        cin >> n >> m >> k;
        vector<int> left(n), right(m);
        for (int i = 0; i < n; i++)
            cin >> left[i];
        for (int i = 0; i < m; i++)
            cin >> right[i];
        for (int i = 0; i < n; i++)
            for (int j = 0; j < m; j++)
                sum += (left[i] + right[j] <= k);
        cout << sum << '\n';
    }
}

File: Python/A/1941A-rudolfandticket.py
t = int(input())
for _ in range(t):
    n, m, k = map(int, input().split())
    left = list(map(int, input().split()))
    right = list(map(int, input().split()))
    sum = 0
    for i in range(n):
        for j in range(m):
            sum += left[i] + right[j] <= k
    print(sum)
